### ISSUE TITLE
"Check direction of streams" crashes

### ISSUE THREAD

**Issue by Mathias-S** [JOSM/kartverketimport#2]
When running the tool "More tools -> Check direction of streams" and there are streams to be checked (i.e. the plugin doesn't produce the message "No more directions to check"), the plugin crashes, and produces the following stack trace:
```
java.lang.IllegalArgumentException: No icon provided
	at org.openstreetmap.josm.gui.SideButton.<init>(SideButton.java:43)
	at org.openstreetmap.josm.plugins.kartverket.CheckDirectionDialog.build(CheckDirectionDialog.java:59)
	at org.openstreetmap.josm.plugins.kartverket.CheckDirectionDialog.<init>(CheckDirectionDialog.java:38)
	at org.openstreetmap.josm.plugins.kartverket.actions.CheckDirectionAction.nextWay(CheckDirectionAction.java:63)
	at org.openstreetmap.josm.plugins.kartverket.actions.CheckDirectionAction.actionPerformed(CheckDirectionAction.java:47)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(Unknown Source)
...
```

Tested on Java 1.8.0_181 and 10.0.2 on Windows 10 and Ubuntu 18.04.1 (clean install of JOSM 14026, no other plugins except utilsplugin2).

**Comment by Mathias-S** [comment]
This bug was caused by https://github.com/openstreetmap/josm/commit/4635423cedfb6a1d0ad142cb7b2d8019e06382dc. I wrote a fix in pull request #3.

**Comment by don-vip** [comment]
Thanks for the PR, I'll make the release tomorrow.
